frontend/test/metabase/scenarios/alert/alert_auth.cy.spec.js

```diff
@@ -0,0 +1,108 @@
+import {
+  restore,
+  signInAsAdmin,
+  signInAsNormalUser,
+  setupLocalHostEmail,
+  createBasicAlert,
+} from "../../../__support__/cypress";
+// Port from alert.e2e.spec.js
+
+describe("scenarios > alert > auth for alerts", () => {
+  before(() => {
+    restore();
+    signInAsAdmin();
+
+    // Setup email
+    // NOTE: Must run `python -m smtpd -n -c DebuggingServer localhost:1025` before these tests
+    signInAsAdmin();
+    cy.visit("/admin/settings/email");
+    setupLocalHostEmail();
+
+    // Create alert as admin
+    cy.visit("/question/1");
+    createBasicAlert("first alert");
+
+    // Create alert as admin that user can see
+    cy.visit("/question/2");
+    createBasicAlert("include normal");
+
+    // Create alert as normal user
+    signInAsNormalUser();
+    cy.visit("/question/3");
+    createBasicAlert();
+  });
+
+  describe("as an admin", () => {
+    beforeEach(signInAsAdmin);
+
+    it("should let you see all created alerts", () => {
+      cy.request("/api/alert").then(response => {
+        expect(response.body).to.have.length(3);
+      });
```

Comment:
Do these get listed in the UI? If so, I think it'd be better to check that text from them is visible.

Reply:
They do not get listed in the UI. You would need to go into each individual question to check if each had an alert through the UI.